src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1309,26 +1309,30 @@ private IWhileLoopOperation CreateBoundWhileStatementOperation(BoundWhileStateme
         {
             Lazy<IOperation> condition = new Lazy<IOperation>(() => Create(boundWhileStatement.Condition));
             Lazy<IOperation> body = new Lazy<IOperation>(() => Create(boundWhileStatement.Body));
+            Lazy<IOperation> ignoredCondition = new Lazy<IOperation>(() => null);
```

Comment:
Is there a singleton instance that can be used here?

Reply:
That would create duplicate operation nodes in the tree.

Reply:
This `Lazy<IOperation>` generates a `null` instance.